sass/utilities/mixins.sass

```diff
@@ -173,8 +173,10 @@
 // Responsiveness
 
 $tablet: 769px !default
-$desktop: 1000px !default // 960px container + 40px
-$widescreen: 1192px !default // 1152px container + 40
+// 960px container + 40px
+$desktop: 1000px !default
+// 1152px container + 40
+$widescreen: 1192px !default
```

Comment:
Oh actually, looking into it further, it seems to choke after `!default` specifically https://github.com/sass/sass/issues/326#issuecomment-5272861